### DISCUSSION TITLE
MTEX "mtex/mex"

### DISCUSSION THREAD

**Discussion by Mabroukimedcea** [mtex-toolbox/mtex#1751] [Ask Anything]
Hi, 
Could onaymone please explain to me the phrase below in the file unstallation or what I have to do to take it into account 

thanks so much in adavanced 

-----------------------------------------------------------
Note, that when cloning MTEX the binaries in `mtex/mex` and `mtex/extern/nfft`
are not included but must be copied from an zip-file based installation as
explained above.

**Comment by kilir** [comment]
It measn that you either download a realese and copy the relevant files of to your git clone or you build them yourself, first option most likely being easier.

Copy:
1 download the relase version
2 unzip
3 copy the content of mtex/mex and mtex/extern/nfft
4 change to your git clone and paste in the equivalent locations

Cheers,
Rüdiger

**Reply by Mabroukimedcea** [reply]
Thanks againa and last question please , 
I saw your answer and one more question, please, as i can not change my glibc, could you tell me please how to compile the nfft without vectorization, thank you again 

Mohamed  
-------------------------------------------------------------------------------------------------------
See here: https://groups.google.com/d/msg/mtexmail/0LkUv2XiPJI/76N4CydxAwAJ
If you can't change to a newer glibc, you might have to compile nfft for this setup without vectorization/openMP support, which will most likely be slower in some tasks.

------------------------------------------------------------------------------------------------

**Comment by Mabroukimedcea** [comment]
Thanks you for the explanation

Le jeu. 2 mars 2023 à 17:10, Rüdiger Kilian ***@***.***> a
écrit :
